### ISSUE TITLE
Choose which interfaces Repository will extend

### ISSUE THREAD

**Issue by tubicstefan** [cmeza20/spring-data-generator#17]
There is a need for functionality which will allow Repository to extend a couple of interfaces that we choose, for example QuerydslPredicateExecutor that is useful for creating dynamic queries.

**Comment by cmeza20** [comment]
Hi @stubic 

In the next version we'll consider the interfaces implementation by election.

Regards

**Comment by jzlima** [comment]
i'll be waiting for this

**Comment by cmeza20** [comment]
Hi @stubic @jzlima 

The functionality was published in version 1.1.7 #19 

```
<dependency>
  <groupId>com.cmeza</groupId>
  <artifactId>spring-data-generator</artifactId>
  <version>1.1.7</version>
</dependency>
```
Add the following configuration, all Repositories extend the interfaces that we choose
```
<additional-extends>
         <param>org.springframework.data.querydsl.QuerydslPredicateExecutor</param>
</additional-extends>
```

Best regards
Carlos Meza
